Adyen/Core/Components/Base/PresentableComponentWrapper.swift

```diff
@@ -24,7 +24,7 @@ public final class PresentableComponentWrapper: PresentableComponent,
     /// The wrapped component.
     public let component: Component
     
-    public let requiresModalPresentation: Bool = true
+    public var requiresModalPresentation: Bool = true
```

Comment:
Why this change is needed?

Reply:
To hide the  topbar/toolbar from the Twint app chooser VC UI.